src/spaceone/inventory/service/cloud_service_service.py

```diff
@@ -454,101 +454,85 @@ def _make_cloud_service_type_key(resource_data):
     def _make_region_key(resource_data, provider):
         return f'{resource_data["domain_id"]}.{provider}.{resource_data["region_code"]}'
 
-    def _change_metadata_path(self, metadata):
-        plugin_id = self.transaction.get_meta('plugin_id')
-
-        if plugin_id:
+    @staticmethod
+    def _convert_metadata(metadata, provider):
+        if provider:
             metadata = {
-                plugin_id: copy.deepcopy(metadata)
+                provider: copy.deepcopy(metadata)
             }
         else:
             metadata = {
-                'MANUAL': copy.deepcopy(metadata)
+                'custom': copy.deepcopy(metadata)
             }
 
         return metadata
 
-    def _get_collection_info(self, collection_info=None):
-        if collection_info is None:
-            collection_info = {}
+    def _get_collection_info(self, provider, collections: CollectionInfo = None):
+        if collections is None:
```

Comment:
change a syntax

`collections = collections or []`

Reply:
have reflected thank you.